Gemfile

```
@@ -10,7 +10,19 @@ elsif ENV['RAILS_VERSION']
   gem "rails", "~> #{ENV['RAILS_VERSION']}"
```

Related to https://github.com/gocardless/statesman/pull/548/files#r1945092519, we should have a default version of Rails here for when `RAILS_VERSION` is not set, as we want to ensure there's always a Rails development dependency, but not necessarily a runtime dependency
```suggestion
  gem "rails", "~> #{ENV['RAILS_VERSION']}"
else
  gem "rails", ">= 7.0"
```